src/pages/BrowseSchemes.svelte

```diff
@@ -2,6 +2,7 @@
   // @ts-ignore no declarations
   import { initAll } from "govuk-frontend";
   import "../style/main.css";
+  import AppVersion from "lib/browse/AppVersion.svelte";
```

Comment:
Should this be on all the pages? I know what we currently have is specific to browse schemes but it feels like we might want something similar in other places?

Reply:
PTAL -- changed a few things about detecting the environment, mentioning it on multiple pages, etc